test/units/testsuite-75.sh

```diff
@@ -76,9 +79,7 @@ if knotc zone-get test. onlinesign.test. ds | grep .; then
     # Drop any old DS records, if present (e.g. on test re-run)
     knotc zone-unset test. onlinesign.test. ds
 fi
-knotc zone-commit test.
 # Propagate the new DS records
-knotc zone-begin test.
```

Comment:
The change in the third commit should be moved to the first commit.

Reply:
Whoops, indeed, thanks.